projects.html

```diff
@@ -41,100 +42,73 @@
 <li><a href="education.html">Education</a></li>
 </ul>
 
-  <div class="main">
-    <div class="section">
-      <div class="heading">Projects</div>
-      <div class="sub-heading">
-        "When something is important enough, you do it even if the odds are not
-        in your favor"
+<div class="container" style="width: 100%;">
+<section style=" height: 70vh;" class="animate__animated animate__fadeInDown">
+  <div class="px-3 pt-5">
+    <div class="row text-dark">
+      <div
+        class="col-md-6 d-flex mt-5 flex-column text-center justify-content-center animate__animated animate__fadeInLeft animate__delay-0.5s">
+        <h2 style="font-size: 4em;font-weight: bolder;" class="mb-0">Projects</h2>
+        <p class="font-weight-bolder text-muted" style="font-size: 1.2em; letter-spacing: 2px;">
+          My projects makes use of vast variety of latest technology tools. My best experience is to create Data Science projects and deploy them to web applications using cloud infrastructure.
```

Comment:
I feel the text is very clumsy, Align it on one side or ask @anushbhatia for the same, I am not an expert at Design. But it looks clumsy.

Reply:
okay then :)